features/environment.py

```diff
@@ -15,7 +15,7 @@ def before_all(context):
     context.wait_seconds = WAIT_SECONDS
     # Select either Chrome or Firefox
     if 'firefox' in DRIVER:
-        context.driver = get_firefox()
+        context.diver = get_firefox()
```

Comment:
For some reason, I can't make a comment on line 9

Change this back to context.driver

Replace line 9 with
DRIVER = getenv('DRIVER', 'chrome').lower()

Try that?